janusgraph-server/src/main/java/org/janusgraph/graphdb/database/management/CfgManagementSystem.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.database.management;
+
+import org.janusgraph.core.schema.JanusGraphManagement;
+import org.janusgraph.graphdb.database.StandardJanusGraph;
+import org.janusgraph.graphdb.database.serialize.StandardSerializer;
+import org.janusgraph.graphdb.database.serialize.attribute.EnumSerializer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.concurrent.Callable;
+
+public class CfgManagementSystem {
+    private final StandardJanusGraph graph;
+    private final CfgManagementLogger managementLogger;
+
+    public CfgManagementSystem(StandardJanusGraph graph) {
+        this.graph = graph;
+        this.managementLogger = (CfgManagementLogger) graph.getManagementLogger();
+    }
+
+    /**
+     * Upon the open managementsystem's commit, this graph will be asynchronously evicted from the cache on all JanusGraph nodes in your
+     * cluster, once there are no open transactions on this graph on each respective JanusGraph node
+     * and assuming each node is correctly configured to use the {@link org.janusgraph.graphdb.management.JanusGraphManager}.
+     */
+    public void evictGraphFromCache() {
```

Comment:
Is there any specific reason for moving `evictGraphFromCache` from `ManagementSystem`? It essentially does the same thing as in `ManagementSystem` and doesn't add any additional funtionalies.